code/obj/item/gun/kinetic.dm

```diff
@@ -1692,3 +1699,9 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 			icon_state="guncase"
 		else
 			icon_state="secure"
+
+#undef MELEE_DMG_PISTOL
+#undef MELEE_DMG_REVOLVER
+#undef MELEE_DMG_SMG
+#undef MELEE_DMG_RIFLE
+#undef MELEE_DMG_LARGE
```

Comment:
Actually, I'm gonna be picky here and ask you to just move the defines to somewhere in `/_std/defines`, because there are a few other places where kinetic firearms are defined e.g.: `tarmstuff.dm`, and those should also be unified at some point